Tasks/UPackCommandV0/Common/ArtifactToolRunner.ts

```diff
@@ -0,0 +1,68 @@
+import * as auth from "./Authentication";
+import * as tl from "vsts-task-lib/task";
+let fs = require("fs");
+let os = require("os");
+import child = require("child_process");
+import stream = require("stream");
+import {IExecOptions, IExecSyncResult} from "vsts-task-lib/toolrunner";
+
+export interface IArtifactToolOptions {
+    artifactToolPath: string;
+    feedId: string;
+    accountUrl: string;
+    packageName: string;
+    packageVersion: string;
+    authInfo: auth.UPackExtendedAuthInfo;
+}
+
+function getOptions(): IExecOptions{
+    let result: IExecOptions = <IExecOptions>{
+        cwd: process.cwd(),
+        env: process.env,
```

Comment:
Can you clone the environment and inject the PAT only into the artifacttool process environment, instead of the current process environment?

Reply:
The artifact tool process env and current process env is same right now. Is there any benefit of creating a clone?